iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/CreateVoltageLevelTopology.java

```diff
@@ -156,29 +153,29 @@ public void apply(Network network, boolean throwException, ComputationManager co
                 LOG.warn("Voltage level {} is BUS_BREAKER. Switchkinds is ignored.", voltageLevelId);
             }
             // Create buses
-            createBuses(voltageLevel);
+            createBuses(voltageLevel, namingStrategy);
             // Create switches between buses
-            createBusBreakerSwitches(voltageLevel);
+            createBusBreakerSwitches(voltageLevel, namingStrategy);
         } else {
             // Check switch kinds
             if (!checkSwitchKinds(switchKinds, sectionCount, reporter, throwException)) {
                 return;
             }
             // Create busbar sections
-            createBusbarSections(voltageLevel);
+            createBusbarSections(voltageLevel, namingStrategy);
             // Create switches
-            createSwitches(voltageLevel);
+            createSwitches(voltageLevel, namingStrategy);
         }
         LOG.info("New symmetrical topology in voltage level {}: creation of {} bus(es) or busbar(s) with {} section(s) each.", voltageLevelId, alignedBusesOrBusbarCount, sectionCount);
         createdNewSymmetricalTopology(reporter, voltageLevelId, alignedBusesOrBusbarCount, sectionCount);
     }
 
-    private void createBusbarSections(VoltageLevel voltageLevel) {
+    private void createBusbarSections(VoltageLevel voltageLevel, NamingStrategy namingStrategy) {
         int node = 0;
         for (int sectionNum = lowSectionIndex; sectionNum < lowSectionIndex + sectionCount; sectionNum++) {
             for (int busbarNum = lowBusOrBusbarIndex; busbarNum < lowBusOrBusbarIndex + alignedBusesOrBusbarCount; busbarNum++) {
                 BusbarSection bbs = voltageLevel.getNodeBreakerView().newBusbarSection()
-                        .setId(busOrBusbarSectionPrefixId + SEPARATOR + busbarNum + SEPARATOR + sectionNum)
+                        .setId(namingStrategy.getBusbarId(busOrBusbarSectionPrefixId, switchKinds, busbarNum, sectionNum))
```

Comment:
Same here, the goal of the naming strategy would be not to have to fill these ID for each modification

Reply:
Same question : do we really want to delete the possibility for the user to specify a prefix for those elements ?